Common/TaskCreator.cs

```diff
@@ -5,15 +5,15 @@ namespace Common
 {
     public static class TaskCreator
```

Comment:
Let's change the name of the class to `SubtaskCreator` since this is what it does :smile: The same goes for the file name